org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizard.java

```diff
@@ -56,55 +60,84 @@ public boolean performFinish() {
 		}
 
 		if (!template.isEmpty()) {
```

Comment:
Why this condition? In case no template is selected, then we should invoke `dotnet new` anyway, shouldn't we?